bin/deploy.js

```diff
@@ -16,29 +16,24 @@ const startTime = Date.now();
 const commander = require('commander');
 
 commander
-.arguments('<templateFile> <refocusUrl> <refocusToken>')
-.option('--isPublished <true|false>', 'set isPublished to true|false')
+.arguments('<templateFile> <isPublished> <refocusUrl> <refocusToken>')
 .parse(process.argv);
 
-if (commander.args.length < 3) {
-  console.error('\nError: <templateFile> <refocusUrl> <refocusToken> args are required.\n');
+if (commander.args.length < 4) {
+  console.error('\nError: <templateFile> <isPublished=(true|false)> <refocusUrl> <refocusToken> args are required.\n');
   process.exit(1);
 }
 
-if (!commander.hasOwnProperty('isPublished')) {
-  console.error('\nError: --isPublished <true|false> is required.\n');
-  process.exit(1);
-}
+let [templateFile, isPublished, refocusUrl, refocusToken] = commander.args;
 
-if (!['true', 'false'].includes(commander.isPublished.toLowerCase())) {
-  console.error('\nError: --isPublished <true|false> is required.\n');
+if (!['isPublished=true', 'isPublished=false'].includes(isPublished)) {
+  console.error('\nError: isPublished must be "isPublished=<true|false>".\n');
```

Comment:
the wording of this error message seems awkward but I'm not seeing an obvious way to re-word it